.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,371 @@
+and allow the design idea/proposal to evolve into something that we can agree to.
+
+## Styleguides
+
+### Changelog entries
+- [ ] TODO - define how/where changelog entries are managed.
+   a. If fixing a bug, mention the fix in the changelog file. TODO - what file?
+   a. If adding a new feature, mention the enhancement in the changelog file.
+   a. If appropriate, summarize the feature in Readme.txt.
+
+### Git Commit messages
+TODO...
+
+### Coding Guidelines
+TODO...
+
+## Additional Notes
+
+### Issue and Pull Request Labels
+This section lists the labels we use to track and manage issues and pull requests.
+
+GitHub search make it easy to use labels for finding groups of issues or pull requests
+you're interested in.
+For example, you might be interested in beginner issues or perhaps open pull requests which
+haven't been reviewed. To help you find issues and pull requests, each label is listed
+with a search link for finding open items with that label.
+See [other search filters](TODO) which will help you write more focused queries.
+
+We will grow these labels over time as we better understand this process and how to use them.
+Please open an issue if you have suggestions for new labels.
+
+#### Issue Type
+Label name | Search | Description
+-----------|--------|------------
+`bug` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Abug) | Confirmed bugs or reports that are very likey to be bugs.
+`documentation` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Adocumentation) | Related to any type of documentation.
+`enhancement` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Aenhancement) | New feature or requests, including contest nominations.
+`feedback` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Afeedback) | General feedback; more than bug reports and feature requests.
+`question` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Aquestion) | Further information is requested. Questions that are more than bug reports or feature requests.
+`training` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Atraining) | Related to any type of training or presentation material (e.g. club presentation).
+
+#### Issue Attribute
+Label name | Search | Description
+-----------|--------|------------
+`good-first-issue` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Agood-first-issue) | Less complex issues which would be good first issues to work on for users who wanted to contribute to MorseRunner.
+`beginner` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aatom%2Fatom+label%3Abeginner) | Less complex issues which would be good first issues to work on for users who wanted to contribute to MorseRunner.
```

Comment:
Please delete this line.

Reply:
these two lines are pointing to atom project. Please resolve and search for other occurrences.